website/docs/docs/collaborate/data-tile.md

```diff
@@ -0,0 +1,211 @@
+
+<DocCarousel slidesPerView={1}>
+<Lightbox src="/img/docs/collaborate/dbt-explorer/data-tile-iframe.jpg" width="70%" title="Example of embedded iFrame" />
+<Lightbox src="/img/docs/collaborate/dbt-explorer/data-tile-pass.jpg" width="60%" title="Example of passing Data health tile in your dashboard." />
+<Lightbox src="/img/docs/collaborate/dbt-explorer/data-tile-stale.jpg" width="60%" title="Example of stale of degraded Data health tile in your dashboard." />
+</DocCarousel>
+
+
+3. For the job-based exposure tile you can insert these three fields into the following iFrame, and then embed them with your dashboard. The next section will have more details on the job-based exposure tile.
+
+    `<iframe src='https://metadata.YOUR_ACCESS_URL/exposure-tile?name=<exposure_name>&environment_id=<environment_id>&token=<metadata_token>' />`
+
+    *Note, replace the placeholders with your actual values.*
+
+## Job-based data health <Lifecycle status="Legacy"/>
+
+The default experience is the [environment-based data health tile](#view-exposure-in-dbt-explorer) with dbt Explorer.
+
+This section is for legacy job-based data health tiles. If you're using the revamped environment-based exposure tile, refer to the previous section. Expand the following to learn more about the legacy job-based data health tile. However, note that 
+
+<Expandable alt_header="Job-based data health">  
+In dbt Cloud, the [Discovery API](/docs/dbt-cloud-apis/discovery-api) can power dashboard status tiles, which are job-based.  A dashboard status tile is placed on a dashboard (specifically: anywhere you can embed an iFrame) to give insight into the quality and freshness of the data feeding into that dashboard. This is done via dbt [exposures](/docs/build/exposures).
+
+#### Functionality
+The dashboard status tile looks like this:
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/dashboard-status-tiles/passing-tile.jpeg"/>
+
+The data freshness check fails if any sources feeding into the exposure are stale. The data quality check fails if any dbt tests fail. A failure state could look like this:
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/dashboard-status-tiles/failing-tile.jpeg"/>
+
+Clicking into **see details** from the Dashboard Status Tile takes you to a landing page where you can learn more about the specific sources, models, and tests feeding into this exposure.
+
+#### Setup
+First, be sure to enable [source freshness](/docs/deploy/source-freshness) in the job that generates this exposure.
+
+In order to set up your dashboard status tile, here is what you need:
+
+1. **Metadata Only token.**  You can learn how to set up a Metadata-Only token [here](/docs/dbt-cloud-apis/service-tokens).
+
+2. **Exposure name.** You can learn more about how to set up exposures [here](/docs/build/exposures).
+
+3. **jobID.** Remember that you can select your jobId directly from the URL when looking at the relevant job in dbt Cloud.
```

Comment:
Hiya @mirnawong1 

Just a QQ on this - should jobID be two words (job ID)?

Kind Regards